File: sioga/2.Codigo/backend/SiogaApiPublic/Domain/CuentaCorriente.cs
using System;

namespace SiogaApiPublic.Domain
{
    public class CuentaCorriente
    {
        public int CuentaCorrienteId { get; set; }
        public int BancoId { get; set; }
        public int FuenteFinanciamientoId { get; set; }
        public int UnidadEjecutoraId { get; set; }
        public string Codigo { get; set; }
        public string Numero { get; set; }
        public string Denominacion { get; set; }
    }
}

File: recaudacion/2.Codigo/frontend/src/app/core/models/cuentacorriente.ts
export class CuentaCorriente {
    cuentaCorrienteId: number;
    bancoId: number;
    fuenteFinanciamientoId: number;
    unidadEjecutoraId: number;
    codigo: string;
    fecha: any;
    numero: string;
    denominacion: string;
    tipo: string;
    moneda: string;
    observacion: string;
    estado: boolean;
    usuarioCreador: string;
    usuarioModificador: string;
    numeroDenominacion: string;
    index: number;
}

export class CuentaCorrienteFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn: string;
    sortOrder: string;
    unidadEjecutoraId: number;
    codigo: string;
    numero: string;
    denominacion: string;
    estado: boolean;
}

